src/gamsproperties.cpp

```diff
@@ -0,0 +1,96 @@
+#include <QDir>
+#include <QTime>
+#include <QDebug>
+
+#include "gamsproperties.h"
+#include "exception.h"
+#include "option/option.h"
+
+namespace gams {
+namespace studio {
+
+GamsProperties::GamsProperties(FileId origin) : mFileId(origin)
+{
+    // set default path
+    mGamsArgs.insert("lo", "3");
+    mGamsArgs.insert("ide", "1");
+    mGamsArgs.insert("er", "99");
+    mGamsArgs.insert("errmsg", "1");
+    mGamsArgs.insert("pagesize", "0");
+    mGamsArgs.insert("LstTitleLeftAligned", "1");
+}
+
+void GamsProperties::setAndAnalyzeParameters(const QString &inputFile, QList<OptionItem> itemList)
```

Comment:
@rgoltermann This function has some TODOs left. Furthermore, we have to check that we do **not** apply any "space issue" workaround, except for the input file and the things @jparnjai is doing during the options processing.